platform/security/filter/security-filter-csrf/pom.xml

```diff
@@ -70,17 +70,17 @@
                                         <limit implementation="org.codice.jacoco.LenientLimit">
                                             <counter>INSTRUCTION</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.91</minimum>
+                                            <minimum>0.96</minimum>
                                         </limit>
                                         <limit implementation="org.codice.jacoco.LenientLimit">
                                             <counter>BRANCH</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.91</minimum>
+                                            <minimum>1.00</minimum>
```

Comment:
You need a little buffer room here. I don't trust this.... nobody's perfect

Reply:
The `LenientLimit` jacoco implementation adds an automatic buffer to account for floating point variance.

Reply:
Actually, the issue is not a floating point one. The issue is a difference in byte code generated by different versions of compilers or different OS.